locales/lt-LT/messages.po

```diff
@@ -0,0 +1,1833 @@
+msgstr "Vaizdinėje aplinkoje, tekstinės užduotys tipiškai yra tos, kurias kandidatas įrašo.  Pateikties variklis turėtų naudoti reikšmę šios savybės (jeigu ji pateikta), vietoj numatyto teksto, kai tai yra reikalaujama."
+
+msgid "incorrect"
+msgstr "neteisingas"
+
+msgid "Index"
+msgstr "Indeksas"
+
+msgid "Infinity"
+msgstr "Begalybė"
+
+msgid "inline"
+msgstr "įterptas"
+
+msgid "Inline Choice"
+msgstr "Įterptas Pasirinkimas"
+
+msgid "Inline Choice Interaction"
+msgstr "Įterpto Pasirinkimo Užduotis"
+
+msgid "Inline interactions need to be inserted into a text block."
+msgstr "Įterptos užduotis turi būti įdėtos į teksto bloką."
+
+msgid "Interaction Background"
+msgstr "Užduoties Fonas"
+
+msgid "Interaction Properties"
+msgstr "Užduoties Savybės"
+
+msgid "Interpretation"
+msgstr "Interpretacija"
+
+msgid "invalid identifier"
+msgstr "blogas identifikatorius"
+
+msgid "invalid QTI XML"
+msgstr "blogas QTI XML"
+
+msgid "Is this choice the correct response?"
+msgstr "Ar šis pasirinkimas yra teisingas atsakymas?"
+
+msgid "Is this pair the correct response?"
+msgstr "Ar ši pora yra teisingas atsakymas?"
+
+msgid "Item \"%s\" cannot be exported: %s"
+msgstr "Elementas \"%s\" negali būti eksportuotas: %s"
+
+msgid "Item \"%s\" has no xml document"
+msgstr "Elementas \"%s\" neturi xml dokumento"
+
+msgid "Item \"%s\" is not available in any language"
+msgstr "Elementas \"%s\" nepasiekiamas jokia kalba"
+
+msgid "Item \"%s\" is ready to be exported"
+msgstr "Elementas \"%s\" paruoštas būti eksportuotam"
+
+msgid "Item metadata successfully exported."
+msgstr "Elemento meta duomenys sėkmingai išeksportuoti."
+
+msgid "Item metadata with identifier \"%s\" is not valid: "
+msgstr "Elemento meta duomenys su identifikatoriumi \"%s\" nėra teisingi: "
+
+msgid "Item properties"
+msgstr "Elemento savybės"
+
+msgid "Item Properties"
+msgstr "Nuotraukos Savybės"
+
+msgid "Item width"
+msgstr "Elemento plotis"
+
+#, tao-public
+msgid "Items"
+msgstr "Elementai"
+
+msgid "JPEG image"
+msgstr "JPEG paveiksliukas"
+
+msgid "Label"
+msgstr "Pavadinimas"
+
+msgid "Language"
+msgstr "Kalba"
+
+msgid "Large editor"
+msgstr "Didelis redaguoklis"
+
+msgid "Latex"
+msgstr "Latex"
+
+msgid "LaTeX"
+msgstr "LaTeX"
+
+msgid "leave empty"
+msgstr "palikti tuščią"
+
+msgid "Left"
+msgstr "Kairė"
+
+msgid "left"
+msgstr "kairė"
+
+msgid "Left bracket"
+msgstr "Kairys skliaustas"
+
+msgid "Length"
+msgstr "Ilgis"
+
+msgid "Level"
+msgstr "Lygis"
+
+msgid "Lines"
+msgstr "Linijos"
+
+msgid "List Style"
+msgstr "Sąrašo Stilius"
+
+msgid "Ln"
+msgstr "Ln"
+
+msgid "Log"
+msgstr "Žurnalas"
+
+msgid "Loop"
+msgstr "Ciklas"
+
+msgid "Lower Bound"
+msgstr "Apatinė Riba"
+
+msgid "Lower than or equal"
+msgstr "Mažiau negu arba lygu"
+
+msgid "Malformed XML[%s]:\n"
+"%s"
+msgstr "Klaidingas XML[%s]:\n"
+"%1"
```

Comment:
Why it's `%1` instead `%s`

Reply:
![image](https://user-images.githubusercontent.com/18699340/66566909-c7a63800-eb6e-11e9-823f-d3b9e9c41770.png)

Reply:
you right, it's need %s, i'll fix it

Reply:
fixed